jwst/pipeline/tests/test_calwebb_spec3_mir_mrs_dither.py

```diff
@@ -67,10 +65,8 @@ def test_run_extract_1d_only(mk_tmp_dirs):
     """
     tmp_current_path, tmp_data_path, tmp_config_path = mk_tmp_dirs
 
-    asn_path = update_asn_basedir(
-        path.join(DATAPATH, 'cube_build_4dither_495_asn.json'),
-        root=path.join(DATAPATH, 'level2b')
-    )
+    asn_path = path.join(DATAPATH, 'cube_build_4dither_495_asn.json')
```

Comment:
I am not familiar with `update_asn_basedir` but these two changes are different. Is this intentional? I mean `leve2b` is not  in the path here.

Reply:
👍 but now curious as to the passing test...